### PR TITLE
Adds various `lamp` icons

### PR DESCRIPTION
**Pull request by karsa-mistmere**
## Icon Request

* Icon name: lamps
* Use case: for smart home or booking apps
* Originates from: #683
* Screenshots of similar icons:
image

image

### CODE REVIEW

File: tags.json [hunk 1]
```diff
@@ -1647,6 +1647,35 @@
     "settings",
     "mouse"
   ],
+  "lamp": [
+    "lighting",
+    "household"
+  ],
```
Comment (ericfennis) [inline tags.json:1653]:
```suggestion
    "household",
    "home",
    "furniture"
  ],
```

File: tags.json [hunk 2]
```diff
@@ -1647,6 +1647,35 @@
...
+  "lamp-ceiling": [
+    "lighting",
+    "household"
```
Comment (ericfennis) [inline tags.json:1656]:
```suggestion
    "household",
    "home",
    "furniture"
```

File: tags.json [hunk 3]
```diff
@@ -1647,6 +1647,35 @@
...
+  ],
+  "lamp-desk": [
+    "lighting",
+    "household",
+    "office",
+    "desk"
```
Comment (ericfennis) [inline tags.json:1662]:
```suggestion
    "desk",
    "home",
    "furniture"
```

File: tags.json [hunk 4]
```diff
@@ -1647,6 +1647,35 @@
...
+  ],
+  "lamp-floor": [
+    "lighting",
+    "household",
+    "floor"
```
Comment (ericfennis) [inline tags.json:1667]:
```suggestion
    "floor",      
    "home",
    "furniture"
```

File: tags.json [hunk 5]
```diff
@@ -1647,6 +1647,35 @@
...
+  ],
+  "lamp-wall-down": [
+    "lighting",
+    "household",
+    "wall"
```
Comment (ericfennis) [inline tags.json:1672]:
```suggestion
    "wall",
    "home",
    "furniture"
```

File: tags.json [hunk 6]
```diff
@@ -1647,6 +1647,35 @@
...
+  ],
+  "lamp-wall-up": [
+    "lighting",
+    "household",
+    "wall"
```
Comment (ericfennis) [inline tags.json:1677]:
```suggestion
    "wall",
    "home",
    "furniture"
```

**Review by ericfennis — CHANGES_REQUESTED** [review]
Some additional tags.

Can you maybe add the `.idea` directory to the `.gitignore` file and remove these files?

**Comment by karsa-mistmere** [comment]
> Some additional tags.
> 
> Can you maybe add the `.idea` directory to the `.gitignore` file and remove these files?

Sure. I've been tiptoeing around that `.idea` directory for a while now.
I've also added the tags as suggested.

**Pull request merged**